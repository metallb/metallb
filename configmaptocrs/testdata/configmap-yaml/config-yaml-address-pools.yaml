apiVersion: v1
kind: ConfigMap
metadata:
  namespace: metallb-system
  name: config
data:
  config: |
    address-pools:
    - name: my-ip-space-bgp
      protocol: bgp
      addresses:
      - 198.51.100.0/24
      - fc00:f853:0ccd:e799::/124
      avoid-buggy-ips: true
      auto-assign: true
    - name: my-ip-space-l2
      protocol: layer2
      addresses:
      - 192.168.1.240/28
