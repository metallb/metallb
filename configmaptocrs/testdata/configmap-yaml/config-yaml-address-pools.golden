# This was autogenerated by MetalLB's custom resource generator.
apiVersion: metallb.io/v1beta1
kind: IPAddressPool
metadata:
  creationTimestamp: null
  name: my-ip-space-bgp
  namespace: metallb-system
spec:
  addresses:
  - 198.51.100.0/24
  - fc00:f853:0ccd:e799::/124
  autoAssign: true
  avoidBuggyIPs: true
status: {}
---
apiVersion: metallb.io/v1beta1
kind: IPAddressPool
metadata:
  creationTimestamp: null
  name: my-ip-space-l2
  namespace: metallb-system
spec:
  addresses:
  - 192.168.1.240/28
status: {}
---
apiVersion: metallb.io/v1beta1
kind: BGPAdvertisement
metadata:
  creationTimestamp: null
  name: bgpadvertisement1
  namespace: metallb-system
spec:
  ipAddressPools:
  - my-ip-space-bgp
status: {}
---
apiVersion: metallb.io/v1beta1
kind: L2Advertisement
metadata:
  creationTimestamp: null
  name: l2advertisement1
  namespace: metallb-system
spec:
  ipAddressPools:
  - my-ip-space-l2
status: {}
---
